NitroxModel/Discovery/InstallationFinders/EnvironmentGameFinder.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace NitroxModel.Discovery.InstallationFinders
+{
```

Comment:
Small code consistency point here: We recently decided to use [file scoped namespace](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces) in new files.

Reply:
Fix it, check again, please